pkg/controller/account/account_controller.go

```diff
@@ -978,6 +978,24 @@ func CreateAccount(reqLogger logr.Logger, client awsclient.Client, accountName,
 	return accountStatus, nil
 }
 
+func ClaimAccount(currentAcctInstance *awsv1alpha1.Account) {
```

Comment:
Could you add the function call to `statusUpdate` into this function as this is generally the next call made after claiming the account. The we could just return the result from here

Reply:
Ok updated it, so the ClaimAccount will also do the statusUpdate and return the error (if any).